src/main/java/binaries/jesus/BinaryArray.java

```diff
@@ -18,6 +18,37 @@ public BinaryArray(int[] binaries) {
      * Returns the decimal numeric value equivalent to the list of binaries stored
      */
     public int ConvertToDecimalInt() {
-        throw new UnsupportedOperationException("Not supported yet.");
+        //throw new UnsupportedOperationException("Not supported yet.");
+        if (binaries==null) {
+            return 0;
+        }
+
+        int [] arrPow=new int [binaries.length];
+        for (int i = 0; i < binaries.length; i++) {
+            arrPow[i]= (int) pow(binaries)[i];
+        }
+        return sumPow(arrPow,binaries);
+    }
+
+    public int sumPow(int [] arrPow,int arrBinary[]){
+        int sum=0;
+        for (int i = 0; i <arrBinary.length ; i++) {
+            if (arrBinary[i]==1){
+                sum+=arrPow[i];
+            }
+        }
+        return sum;
     }
+
+    public  double [] pow(int arrayBinary[]){
+        double [] arrPow= new double[arrayBinary.length];
+        int suma=0;
```

Comment:
spaces missing before and after `=`. Please also check this in other parts of your code and fix.